### ISSUE TITLE
Max Dex via Effects doesn't modify skill totals

### ISSUE THREAD

**Issue by fougerec** [MetaMorphic-Digital/torgeternity#67]
Using an active effect to downgrade Dexterity via Effects doesn't impact Dexterity based skills.  I have the Item set with a Max Dex of 10.  The 10 shows up in the attributes (down from 12) but the Dodge skills still reflect the original base.
![issue](https://user-images.githubusercontent.com/84296981/118419464-71767080-b692-11eb-95f2-c485e96cd7d7.png)

**Comment by pemerson71** [comment]
I've simply been working with inputting the effects affected by the limitations in the Active Effects tab for each item, though I really need to go back and look at the Equipment for the limitations and how they affect other skills.  Fortunately, there are only 70 skills that are affected, fifteen for Dexterity, I think, so it's not been too onerous.

However, Matt, I know ya'll have been talking about how to make stuff like this happen for skills and defenses and, honestly, it's pretty important to the game and could, if GMs are unwilling to make all the entries and changes like I have made, they may actually leave the game, itself.  I have ALL of the books from Year One and I have been working, for three years, to get all the items entered, having to go back multiple times to make changes again-and-again, whether for changes to Foundry or to the TGS.  I understand the need for the program to be able to grow, but like Foundry there's no real installer/updater for using all of these changes.  It's all copy-paste at this point and, if something is not set up the way we were not told to do, like what happened two months ago when I tried to update from Foundry v10 to v11, everything goes haywire, and I am not a programmer.  Truthfully, I've begun feeling as though the Foundry team really doesn't care about the folks who want to use their service but are unable to program for themselves.

Anyway, the point is we either need the sourcebook items updated with all the Effects already programmed on, or we need to have a system where bonuses, penalties, and limitations trickle down properly.  It took me three years to realize my bonsues, penalties, and limitations were not trickling to skills appropriately, and then it took me two months to go back on all sorts of items and put in the appropriate Effects.  So, my game ran without containment for three years, lending more to the success of my Players than I can quantify, just now.  Things are so much more even now that I've been through those items, I feel like I can give my Friends a truly fair game, as opposed to them running roughshod over everything I've created.

This is important.
